CI process broken

**Describe the bug**
Recently, the `Test / Format` CI broken. The detailed log is:

```
INFO     -  Cleaning site directory
INFO     -  Building documentation to directory: /home/runner/work/codalab-worksheets/codalab-worksheets/site
ERROR    -  Error reading page 'CLI-Reference.md': format() argument must be a formatter instance, not a class
Traceback (most recent call last):
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/pygments/__init__.py", line 61, in format
    formatter.format(tokens, realoutfile)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/pygments/formatter.py", line 94, in format
    return self.format_unencoded(tokensource, outfile)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/pygments/formatters/html.py", line 981, in format_unencoded
    source = self.wrap(source)
TypeError: wrap() missing 1 required positional argument: 'outfile'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "venv/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/mkdocs/__main__.py", line 187, in build_command
    build.build(config.load_config(**kwargs), dirty=not clean)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/mkdocs/commands/build.py", line 292, in build
    _populate_page(file.page, config, files, dirty)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/mkdocs/commands/build.py", line 174, in _populate_page
    page.render(config, files)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/mkdocs/structure/pages.py", line 174, in render
    self.content = md.convert(self.markdown)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/markdown/core.py", line 268, in convert
    newRoot = treeprocessor.run(root)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/pymdownx/highlight.py", line 483, in run
    code_block_count=self.ext.pygments_code_block
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/pymdownx/highlight.py", line 381, in highlight
    code = highlight(src, lexer, formatter)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/pygments/__init__.py", line 82, in highlight
    return format(lex(code, lexer), formatter, outfile)
  File "/home/runner/work/codalab-worksheets/codalab-worksheets/venv/lib/python3.6/site-packages/pygments/__init__.py", line 69, in format
    raise TypeError('format() argument must be a formatter instance, '
TypeError: format() argument must be a formatter instance, not a class
Error: Process completed with exit code 1.
0s

```

--- a/requirements.txt
+++ b/requirements.txt
@@ -14,6 +14,7 @@ six==1.15.0
 SQLAlchemy==1.3.19
 watchdog>=2.0.0
 fusepy==2.0.4
+pygments==2.11.2
 python-dateutil==2.8.1
 diffimg==0.2.3
 selenium==3.141.0

